packages/contracts/.prettierrc

```diff
@@ -2,7 +2,7 @@
   "plugins": ["prettier-plugin-solidity"],
   "printWidth": 120,
   "semi": true,
-  "tabWidth": 2,
```

Comment:
you can just set use forge to format solidity code, rather change prettier config